app/templates/plans.hbs

```diff
@@ -1,69 +1,132 @@
-{{#travis-layout layoutName="layouts/center" class="text-grey-800 font-sans"}}
-  <section class="px-4 py-3 mb-16">
-    {{!-- Header section --}}
-    <header class="text-center mb-16">
-      <h1 class="text-6xl font-bold">Built for every team</h1>
-      <p>Set up your first project in just one minute.<br>
-        Your first 100 builds are free! No credit card required.
-      </p>
-    </header>
+<TravisLayout @layoutName="layouts/center">
 
-    <ul class="list--plans">
-      <li class="plan">
-        <p class="plans-price">$69</p>
-        <p class="plans-label plans-timeframe">per month</p>
-        <h2 class="plans-name">Bootstrap</h2>
-        <h3 class="plans-label">Ideal for hobby projects</h3>
-        <ul class="list">
-          <li><span class="list-em">1</span>Concurrent job</li>
-          <li>{{svg-jar 'icon-passed' class="w-4 fill-transparent stroke-current text-green"}} Unlimited build minutes</li>
-          <li>{{svg-jar 'icon-passed' class="w-4"}} Unlimited repositories</li>
-          <li>{{svg-jar 'icon-passed' class="w-4"}} Unlimited collaborators</li>
-        </ul>
-      </li>
-      <li class="plan--important">
-        <p class="plans-price">$129</p>
-        <p class="plans-label plans-timeframe">per month</p>
-        <h2 class="plans-name">Startup</h2>
-        <h3 class="plans-label">Best for small teams</h3>
-        <ul class="list">
-          <li><span class="list-em">2</span>Concurrent jobs</li>
-          <li>{{svg-jar 'icon-passed' class="w-4"}} Unlimited build minutes</li>
-          <li>{{svg-jar 'icon-passed' class="w-4"}} Unlimited repositories</li>
-          <li>{{svg-jar 'icon-passed' class="w-4"}} Unlimited collaborators</li>
-        </ul>
-        <div class="align-center">
-          <a href="https://travis-ci.com/signin" class="plans-button--blue" {{action 'gaCta' 'plans-table'}}>Start Trial</a>
-        </div>
-      </li>
+  {{!-- Header section --}}
+  <UiKit::Box
+    @tag='section'
+    @margin={{hash top=3 bottom=16}}
+    @textAlign='center'
+  >
+    <UiKit::Font @tag='h1' @size='6xl' @weight='bold'>
+      Built for every team
+    </UiKit::Font>
+    <p>
```

Comment:
Would it make sense to wrap this text with `UiKit::Font` as well, instead of `p` tag? The `p` tag may have some browser default styling and break the layout in some conditions.

Reply:
Yeah maybe so, change made :)